pkgs/development/tools/dotnet/runtime/2.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchurl
+, autoPatchelfHook
+, curl
+, libkrb5
+, libunwind
+, lttng-ust
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dotnet-runtime";
+  version = "2.2.7";
```

Comment:
Can this be `2.1`? because IIUC `2.1` is a TLS release, and `2.2` is going to reach EOL in December 23, 2019 https://dotnet.microsoft.com/download/dotnet-core. Besides that, LGTM

Reply:
I'm actually considering closing this. #73262 fixes a lot more issues